cdk/lib/auth-stack.ts

```diff
@@ -127,7 +127,12 @@ export class AuthStack extends cdk.Stack {
       new iam.PolicyStatement({
         effect: iam.Effect.ALLOW,
         actions: ["s3:GetObject"],
-        resources: [bucketArn.concat("/public/*.json")],
+        resources: [
+          bucketArn.concat("/public/*.json"),
+          bucketArn.concat("/public/*.png"),
+          bucketArn.concat("/public/*.jpg"),
```

Comment:
do we need to add `jpeg` as well?

Reply:
> do we need to add `jpeg` as well?

Jpegs work. I tested all 4 types. I could add one more line if you want to be more explicit. But I think jpg allows both jpg and jpeg.

Reply:
I don't think it will work if a file is named `banana.jpeg`. The IAM policy matches the exact name of the object including the extension.